runtime/common/src/lib.rs

```diff
@@ -548,16 +687,17 @@ macro_rules! impl_config_traits {
             type LeaveCandidatesDelay = LeaveCandidatesDelay;
             type LeaveDelegatorsDelay = LeaveDelegatorsDelay;
             type MaxBottomDelegationsPerCandidate = MaxBottomDelegationsPerCandidate;
-            type MaxTopDelegationsPerCandidate = MaxTopDelegationsPerCandidate;
+            type MaxCandidates = MaxCandidates;
             type MaxDelegationsPerDelegator = MaxDelegationsPerDelegator;
+            type MaxTopDelegationsPerCandidate = MaxTopDelegationsPerCandidate;
+            type MaxOfflineRounds = MaxOfflineRounds;
             type MinBlocksPerRound = MinBlocksPerRound;
-            type MinCandidateStk = MinCollatorStk;
-            type MinCollatorStk = MinCollatorStk;
-            type MinDelegation = MinDelegatorStk;
-            type MinDelegatorStk = MinDelegatorStk;
+            type MinCandidateStk = MinCandidateStk;
+            type MinDelegation = MinDelegation;
             type MinSelectedCandidates = MinSelectedCandidates;
             type MonetaryGovernanceOrigin = EnsureRoot<AccountId>;
             type OnCollatorPayout = ();
+            type OnInactiveCollator = OnInactiveCollator;
```

Comment:
```rust
pub struct PayoutCollatorOrOrbiterReward;
impl pallet_parachain_staking::PayoutCollatorReward<Runtime> for PayoutCollatorOrOrbiterReward {
	fn payout_collator_reward(
		for_round: pallet_parachain_staking::RoundIndex,
		collator_id: AccountId,
		amount: Balance,
	) -> Weight {
		let extra_weight = if MoonbeamOrbiters::is_orbiter(for_round, collator_id) {
			MoonbeamOrbiters::distribute_rewards(for_round, collator_id, amount)
		} else {
			ParachainStaking::mint_collator_reward(for_round, collator_id, amount)
		};

		<Runtime as frame_system::Config>::DbWeight::get()
			.reads(1)
			.saturating_add(extra_weight)
	}
}

pub struct OnInactiveCollator;
impl pallet_parachain_staking::OnInactiveCollator<Runtime> for OnInactiveCollator {
	fn on_inactive_collator(
		collator_id: AccountId,
		round: pallet_parachain_staking::RoundIndex,
	) -> Result<Weight, DispatchErrorWithPostInfo<PostDispatchInfo>> {
		let extra_weight = if !MoonbeamOrbiters::is_orbiter(round, collator_id.clone()) {
			ParachainStaking::go_offline_inner(collator_id)?;
			<Runtime as pallet_parachain_staking::Config>::WeightInfo::go_offline(
				pallet_parachain_staking::MAX_CANDIDATES,
			)
		} else {
			Weight::zero()
		};

		Ok(<Runtime as frame_system::Config>::DbWeight::get()
			.reads(1)
			.saturating_add(extra_weight))
	}
}
```

This is what I have found for moonbeam. I need to double check if we implemented the correct behaviour.